File: AdventOfCode/AdventOfCode/2020/Day 1/Day1Code.R
i = 1
while (i <= 200){
  a = Day1input[i, 1]
  j = i + 1
  while (j <= 200){
    b = Day1input[j, 1]
    k = j + 1
    while (k <= 200){
      c = Day1input[k, 1]
      if(a + b + c == 2020){
        print(i)
        print(j)
        print(k)
      }
      k = k + 1
    }
    j = j + 1
  }
  i = i + 1
}

File: AdventOfCode/2021/Day 1/Day1Code.py
def reader():
	f = open("Day1input.txt", 'r').read()
	f = f.split('\n')
	f = f[:-1]
	return f

def part1():
	f = reader()
	count = 0
	for i in range(len(f) - 1):
		if(int(f[i+1]) > int(f[i])):
			count+=1
	print(count)

def part2():
	f = reader()
	count = 0
	for i in range(len(f) - 3):
		if(int(f[i+1]) + int(f[i+2]) + int(f[i+3]) > int(f[i]) + int(f[i+1]) + int(f[i+2])):
			count+=1
	print(count)

part1()
part2()
